src/makeCalculator.js

```diff
@@ -4,7 +4,33 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calculator = {
+    result: 0,
+    add(value, currantResult) {
```

Comment:
The parameter `currantResult` seems to be a misspelling. It should be `currentResult` for clarity and correctness.